sig-cloud-provider/CONTRIBUTING.md

```diff
@@ -0,0 +1,213 @@
+# Contributing
+
+Welcome to the Kubernetes SIG Cloud Provider contributing guide.  We are excited
+about the prospect of you joining our community!
+
+## Before You Begin
+
+We strongly recommend you to understand the main [Kubernetes Contributor
+Guide](http://git.k8s.io/community/contributors/guide) and adhere to the
+contribution rules (specially signing the CLA).
+
+You can also check the [Contributor Cheat
+Sheet](/contributors/guide/contributor-cheatsheet/), with common resources for
+existing developers.
+
+Read the [developer guide].
```

Comment:
either this should be `[development guide]` or the link should be renamed `[developer guide]`. imo it sounds better as "developer guide".